app/src/main/java/ch/sheremet/dasfoo/rover/android/client/video/VideoFragment.java

```diff
@@ -97,16 +112,20 @@ public void onSurfaceTextureUpdated(final SurfaceTexture surface) {
      * @param v The view that was clicked.
      */
     @Override
-    public void onClick(View v) {
+    public final void onClick(final View v) {
         switch (v.getId()) {
             case R.id.start_video_button:
                 // Start thread reading on server
                 try {
-                    PropertyReader propertyReader = new PropertyReader(getActivity());
-                    Properties properties = propertyReader.getProperties("network.properties");
-                    String host = ((MainActivity) getActivity()).getHost();
-                    int port = ((MainActivity) getActivity()).getPort();
-                    String password = properties.getProperty("password");
+                    // TODO(ksheremet): check errors
+                    final SharedPreferences sharedPreferences =
+                            PreferenceManager.getDefaultSharedPreferences(getActivity());
+                    final String host = sharedPreferences
+                            .getString(Settings.VIDEO_HOST.toString(), "Get host");
```

Comment:
Value to return if this preference does not exist. Maybe is it better to leave it empty?